shell/packages/sandstorm-db/db.js

```diff
@@ -112,6 +112,7 @@ DevApps = new Mongo.Collection("devapps");
 //
 // Each contains:
 //   _id:  The application ID string (as with Packages.appId).
+//   appId: The same as _id, included to mimic Packages.appId.
```

Comment:
This feels odd to me. We're relying on DevApps and Packages having some similar fields, but not all... Would it be cleaner to translate DevApps into Packages at the locations where we want to mix them?